app/controllers/admin/scrambles_controller.rb

```diff
@@ -0,0 +1,95 @@
+# frozen_string_literal: true
+
+module Admin
+  class ScramblesController < AdminController
+    # NOTE: authentication is performed by admin controller
+
+    def new
+      competition = Competition.find(params[:competition_id])
+      round = Round.find(params[:round_id])
+      # Create some basic attributes for that empty scramble.
+      # Using Scramble.new wouldn't work here: we have no idea what the country
```

Comment:
What do you mean by "country" here?

Reply:
Whoops, you caught me in the act of copy-pasting :see_no_evil: